pkg/cmd/logs.go

```diff
@@ -24,17 +22,17 @@ func newLogsCmd(config *config.Config) *LogsCmd {
 	logsCmd.Cmd = &cobra.Command{
 		Use:   "logs",
 		Args:  validators.NoArgs,
-		Short: "Top-level package for logs commands with Stripe.",
-		Long: fmt.Sprintf(`
-The logs package contains the sub-command 'tail', which allows you to tail your API request logs
-in real-time from Stripe.
+		Short: "Top-level command for interacting with API request logs from Stripe. Run `stripe logs --help` to learn more about different logs commands.",
```

Comment:
This is too long, we should keep it to once sentence.

What about "Interact with Stripe request logs"?